collector/internal/controller.go

```diff
@@ -26,6 +28,8 @@ func NewController(cfg *controller.Config,
 		cfg.MonitorInterval, cfg.ProbabilisticInterval)
 
 	rep, err := reporter.NewCollector(&reporter.Config{
+		Name:                     os.Args[0],
```

Comment:
We should fix this to be stable and not the command name which can change, as it ends up as the `Scope` name.

Reply:
I'm not sure why there should be a constant string here. If eBPF based profiling is running as part of some OTel collector, then this OTel collector is the source.
Which constant string do you suggest and why?

Reply:
As this codepath is exclusive to OTel collector, I think the argument around the logic does not apply here.
If someone wants uses a stand alone profiler or even just own reporter, they can set the value to whatever they decide to. This change does not affect them.